roadmap/implementers-guide/src/node/utility/provisioner.md

```diff
@@ -121,6 +169,22 @@ The subsystem should maintain a set of handles to Block Authorship Provisioning
 
 Forward the message to the appropriate Block Authorship Provisioning Job, or discard if no appropriate job is currently active.
 
-## Block Authorship Provisioning Job
-
-Maintain the set of channels to block authors. On receiving provisionable data, send a copy over each channel.
```

Comment:
Nice. Just to make sure, you checked that the job terminology is no longer present in the code? Should we delete the above sections as well in that case?

Reply:
There was some good info in that section to salvage, so I moved it to the protocol section